curriculum/challenges/english/10-coding-interview-prep/the-odin-project/links-and-images-question-e.md

```diff
@@ -0,0 +1,98 @@
+---
+id: 637f702372c65bc8e73dfe32
+videoId: 0xoztJCHpbQ
+title: Links and Images Question E
+challengeType: 11
+dashedName: links-and-images-question-e
+---
+# --description--
+
+Websites would be fairly boring if they could only display text. Luckily HTML provides a wide variety of elements for displaying all sorts of different media. The most widely used of these is the image element.
+
+To display an image in HTML you use the `<img>` element. Unlike the other elements you have encountered, the `<img>` element is self-closing. Empty, self-closing HTML elements do not need a closing tag.
+
+Instead of wrapping content with an opening and closing tag, it embeds an image into the page using a src attribute which tells the browser where the image file is located. The src attribute works much like the href attribute for anchor tags. It can embed an image using both absolute and relative paths.
```

Comment:
```suggestion
Instead of wrapping content with an opening and closing tag, it embeds an image into the page using a `src` attribute which tells the browser where the image file is located. The `src` attribute works much like the `href` attribute for anchor tags. It can embed an image using both absolute and relative paths.
```